src/v/config/node_config.cc

```diff
@@ -44,15 +46,35 @@ node_config::node_config() noexcept
   , rack(
       *this,
       "rack",
-      "Rack identifier",
+      "A label that identifies a failure zone. Apply the same label to all "
+      "brokers in the same failure zone. When `enable_rack_awareness` is set "
+      "to `true` at the cluster level, the system uses the rack labels to "
+      "spread partition replicas across different failure zones.",
       {.visibility = visibility::user},
       std::nullopt)
   , seed_servers(
       *this,
       "seed_servers",
       "List of the seed servers used to join current cluster. If the "
-      "seed_server list is empty the node will be a cluster root and it will "
-      "form a new cluster",
+      "`seed_servers` list is empty the node will be a cluster root and it "
+      "will form a new cluster. When `empty_seed_starts_cluster` is `true`, "
+      "Redpanda enables one broker with an empty `seed_servers` list to "
```

Comment:
> `seed_servers`

this isn't a big deal, but these strings really aren't intended to contain markdown, and i'm assuming that is what the backticks are for.